avocado_vt/plugins/vt.py

```diff
@@ -18,20 +18,19 @@
 
 import os
 
+from avocado.core import plugin_interfaces
 from avocado.core.loader import loader
 from avocado.core.plugin_interfaces import CLI
+from avocado.core.settings import settings, Settings
 from avocado.utils import path as utils_path
 
-from virttest import data_dir
-from virttest import defaults
-from virttest import standalone_test
+from virttest import data_dir, defaults, standalone_test
 from virttest.compat import get_settings_value
-from virttest.standalone_test import SUPPORTED_TEST_TYPES
-from virttest.standalone_test import SUPPORTED_LIBVIRT_URIS
+from virttest.standalone_test import (SUPPORTED_LIBVIRT_URIS,
```

Comment:
It's *better* to keep such style changes (an improvement in this case) in a separate commit.

Reply:
Sure I will split the commit